diktat-rules/src/main/resources/rules-config.json

```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "DIKTAT_COMMON",
+    "enabled": true,
+    "configuration": {
+      "isCopyrightMandatory": true,
+      "domainName": "org.cqfn.diktat",
+      "copyrightText": "Copyright (c) Huawei Technologies Co., Ltd. 2012-2020. All rights reserved."
```

Comment:
I think it's a good moment to change this pattern

Reply:
@aktsay6 please create a copy of this file, name it rules-config-huawei.json
and in original file remove all mentions of Huawei. Let's make it like a general example